### ISSUE TITLE
Syntax highlighting fails with double quotes inside triple quotes

### ISSUE THREAD

**Issue by NiceGuyIT** [linux-china/jetbrains-just-plugin#23]
JetBrains Just plugin version: 0.4.1

When using double quotes inside [triple backticks][1], the syntax is incorrect. A picture is worth a thousand words.

![image](https://github.com/linux-china/jetbrains-just-plugin/assets/7763429/3337b187-fe56-4f5e-8036-99cb915db0b3)

Removing the double quotes in the first block makes the syntax highlighter happy!

![image](https://github.com/linux-china/jetbrains-just-plugin/assets/7763429/cecbc76c-f805-458f-8493-6a5b6a93c851)

Here's the text for testing.

```just
set positional-arguments := true
set shell := ['nu', '-c']
set export := true

export SOME_DIR := "some/dir"

export NU_VAR := ```
		$nu
		print $"Hello World"
		let $something = "nothing"
		$nu
	```

# Require a command to be available
[private]
require-command command:
	#!/usr/bin/env nu
	print $" '{{ command }}' "
```

Some of the other issues mention the need for rewriting the lexer and/or token analyzer to fix parsing issues. I understand if that's the case for this issue as well.

Thanks for an awesome plugin!

[1]: https://just.systems/man/en/chapter_33.html?highlight=backtick#command-evaluation-using-backticks

**Comment by linux-china** [comment]
Fixed, and a new version uploaded to JetBrains plugin repository and waiting for approval. 
![Snip20240228_2](https://github.com/linux-china/jetbrains-just-plugin/assets/46711/e2fc34fb-4ca7-49e6-bc76-618ec719d65b)

**Comment by linux-china** [comment]
Fixed with version 0.4.3

**Comment by NiceGuyIT** [comment]
TYVM! Awesome turnaround time.
